### PR TITLE
Handle 404 status for json verify

### PR DESCRIPTION
**Pull request by electricworry**
Gracefully handle 404 response for verify --json-update. If --delete also specified, old json deleted, which leads to its disowned packages being deleted.

### CODE REVIEW

**Review by cooperlees — APPROVED** [review]
Makes sense, if you want bonus point could mock / monkey patch url_fetch to raise a ClientResponseError exception to test the code. All good if you don't have time.

**Comment by electricworry** [comment]
I'll take a look at that. Do I just update this branch and it will automatically get added to the PR?

**Comment by cooperlees** [comment]
> I'll take a look at that. Do I just update this branch and it will automatically get added to the PR?

Yup, sure will. Just `git push origin verify-404`

**Comment by cooperlees** [comment]
@electricworry - isort fails in link. I haven't debugged yet but here is the error:
- https://pastebin.com/Q0twYNg5

**Comment by ichard26** [comment]
The `W0503: Deprecated` message is dealt with in PR #767 :D

File: src/bandersnatch/verify.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 from sys import stderr
 from typing import List, Optional, Set
 from urllib.parse import urlparse
+import aiohttp
```
Comment (cooperlees) [inline src/bandersnatch/verify.py:15]: Pretty sure this just needs to move to make isort happy now.

**Review by cooperlees — CHANGES_REQUESTED** [review]
Please just run tox lint which runs pre-commit and it should make isort happy.

File: src/bandersnatch/verify.py (hunk 1 above)
Comment (electricworry) [reply src/bandersnatch/verify.py:16]: Done with "tox -e lint". Got there in the end. Been a busy few days.

**Comment by cooperlees** [comment]
No worries. Thanks for the fix and learning our testing!

Merging as we know the coverage is missed.

**Pull request merged**